vector/src/main/res/layout/item_live_location_users_bottom_sheet.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
```

Comment:
Maybe we could add `android:background="?selectableItemBackground"` to have ripple effect when clicking on a user item?

Reply:
Nice catch, done.